### DISCUSSION TITLE
Welcome to SOILSENS-V5W Discussions!

### DISCUSSION THREAD

**Discussion by PricelessToolkit** [PricelessToolkit/SOILSENS-V5W#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nihaoavr** [comment]
Hello, what insulation adhesive is used to seal the TMP102 sensor?

**Reply by PricelessToolkit** [reply]
Hi, The best one is "Henkel 4311" but any other UV resin may work if adherence with the solder mask is good.

**Comment by nihaoavr** [comment]
Is the reading still correct after the TMP102 sensor is sealed?

**Reply by PricelessToolkit** [reply]
reaction time is slower but precision is the same.
